src/adapters/in-memory/users/get.ts

```diff
@@ -0,0 +1,7 @@
+import { SqlUser } from "../../../types";
+
+export function get(users: SqlUser[]) {
+  return async (tenant_id: string, id: string): Promise<SqlUser | null> => {
+    return users.find((user) => user.id === id) || null;
```

Comment:
any reason not to do?

```suggestion
    return users.find((user) => user.id === id && user.tenant_id === tenant_id) || null;
```

On the other `get` in-memory adapters you are using `tenant_id`

Reply:
I don't suppose it matters as our tests aren't so complex